bentoml/_internal/io_descriptors/pandas.py

```diff
@@ -16,10 +16,46 @@
 else:
     pd = LazyLoader("pd", globals(), "pandas")
 
+try:
+    from typing import Literal
+except ImportError:  # pragma: no cover
```

Comment:
Does this not get covered by our 3.7 test?

Reply:
yep

Reply:
well usually I don't think we need to write test to cover import

Reply:
no, but in this particular case shouldn't it be covered?